sklearn/metrics/_classification.py

```diff
@@ -2220,28 +2213,11 @@ class are present in `y_true`): both likelihood ratios are undefined.
         "`UndefinedMetricWarning` will always be raised in case of a division by zero "
         "and the value set with the `replace_undefined_by` param will be returned."
```

Comment:
We also talked about expanding this error message to include a hint on how to handle this with `catch_warnings`